File: snippets/php/pause_sending_from_a_subaccount.php
require_once('/path/to/mailchimp-transactional-php/vendor/autoload.php');
function run($subAccount)
{
    try {
        $mailchimp = new MailchimpTransactional\ApiClient();
        $mailchimp->setApiKey("YOUR_API_KEY");
        $response = $mailchimp->subaccounts->pause($subAccount);
        if(isset($response->status)){
            echo "This subaccount is now " . $response->status;
        }else{
            echo "Could not change subaccount status: <br>";
            print_r($response);
        }
    } catch (Error $e) {
        echo 'Error: ', $e->getMessage(), "\n";
    }
}

$subAccount = [
    "id" => "UNIQUE_ACCOUNT_ID",
];
run($subAccount);

File: snippets/python/pause_sending_from_a_subaccount.py
import mailchimp_transactional as MailchimpTransactional
from mailchimp_transactional.api_client import ApiClientError

mailchimp = MailchimpTransactional.Client('YOUR_API_KEY')

def run():
  try:
    response = mailchimp.subaccounts.pause({
      "id": "UNIQUE_ACCOUNT_ID"
    })
    print("This subaccount is now {} ".format(response['status']))
  except ApiClientError as error:
    print('An exception occurred: {}'.format(error.text))

run()